1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -16,11 +16,21 @@
 // pickFromArray([1,2,3,4])     //maybe returns 2
 // pickFromArray(coloursArray)  //maybe returns "#F38630"
 //
+
+const div = document.getElementById("content")
+const button = document.querySelector("#quote-btn")
+const p = document.createElement("p")
+
+
 // You DO NOT need to understand how this function works.
-function pickFromArray(choices) {
-  return choices[Math.floor(Math.random() * choices.length)];
-}
+function pickFromArray() {
+    const displayQuote = quotes[Math.floor(Math.random() * quotes.length)];
+    p.innerText = `${displayQuote.quote} 
```

Comment:
This solution works - but I think the intent was for this function to not be modified; however, I think this might be unclear from the task description. This works though - but it relies on the function doing random generation and DOM updates.